platforms/gke/base/use-cases/inference-ref-arch/terraform/comfyui/copy-checkpoints/cloudbuild.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+steps:
+  - id: 'Create Directories'
+    name: 'ubuntu'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        dirs=(checkpoints loras vae text_encoders diffusion_models clip_vision style_models embeddings diffusers vae_approx controlnet gligen upscale_models hypernetworks photomaker classifiers)
+        for dir in "${dirs[@]}"; do
+            mkdir -p /workspace/comfyui-models/$dir
+            touch /workspace/comfyui-models/$dir/put_here
+        done
+
+  - id: 'Download SDXL Base Model'
+    name: 'gcr.io/cloud-builders/curl'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        curl -L -o /workspace/comfyui-models/checkpoints/sd_xl_base_1.0.safetensors "https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/resolve/main/sd_xl_base_1.0.safetensors?download=true"
+
+  - id: 'Download SDXL Refiner Model'
+    name: 'gcr.io/cloud-builders/curl'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        curl -L -o /workspace/comfyui-models/checkpoints/sd_xl_refiner_1.0.safetensors "https://huggingface.co/stabilityai/stable-diffusion-xl-refiner-1.0/resolve/main/sd_xl_refiner_1.0.safetensors?download=true"
+
+  - id: 'Download LTXV Model'
+    name: 'gcr.io/cloud-builders/curl'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        curl -L -o /workspace/comfyui-models/checkpoints/ltx-video-2b-v0.9.safetensors "https://huggingface.co/Lightricks/LTX-Video/resolve/main/ltx-video-2b-v0.9.1.safetensors?download=true"
+
+  - id: 'Download text encoder'
+    name: 'gcr.io/cloud-builders/curl'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        curl -L -o /workspace/comfyui-models/text_encoders/t5xxl_fp16.safetensors "https://huggingface.co/comfyanonymous/flux_text_encoders/resolve/main/t5xxl_fp16.safetensors?download=true"
+
+  - id: 'Upload Models to GCS'
+    name: 'gcr.io/cloud-builders/gcloud'
+    args:
+      - "storage"
```

Comment:
One thing I noticed is that these were getting mounted as `root:comfyuser` in the container. Not sure if that is because of how they are mounted or how they are copied into GCS.

Reply:
I am guessing it is how they are mounted as gcsfuse runs with the root user ?